### PR TITLE
Added ES document indexing

### PR DESCRIPTION
**Pull request by chentex**
Signed-off-by: Vicente Zepeda Mas <vzepedam@redhat.com>

### Description

After each test is finished if the elastic config is set
the process will read the file and Bulk index all the results to the ES
server

### Fixes

This will render obsolete the hack in the automation scripts, and will change the flow in `benchmark-operator`

### CODE REVIEW

File: config.example.yaml [hunk 1]
```diff
@@ -14,6 +14,11 @@ aws:
     access-key: "ASD7ASFET65FFGHDFFS"
     secret-access-key: "fjhgsadf6#$!@%&/dfghdfgdsdf"
     account-id: "123434565665"
+elastic:
```
Comment (dry923) [inline config.example.yaml:17]: Is it possible to take this in as a single line url that already includes the user/pass like we do with benchmark-operator/wrapper?

Comment (chentex) [reply config.example.yaml:17]: It works also with the URL with user/password included. You can leave omit the user and password configs.

**Comment by dry923** [comment]
cc @venkataani

**Review by mukrishn — APPROVED** [review]
LGTM, @chentex thanks for the explanation on using `opensearch` mod over `elasticsearch`.

**Review by venkataanil — COMMENTED** [review]
LGTM

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**